test-result-summary-client/src/Build/Summary/ResultSummary.jsx

```diff
@@ -93,8 +93,11 @@ export default function ResultSummary() {
                 if (buildName.includes('_smoketests')) {
                     level = 'extended';
                 }
-
                 let jdkVersion, platform, jdkImpl;
+                const parentRegex = /openjdk(\d+)-pipeline/i;
+                const parenttokens =
+                    parentBuildInfo.buildName.match(parentRegex);
+                jdkVersion = parenttokens[1];
```

Comment:
I think we should move this outside of the sdkBuilds.forEach() as the `parentBuildInfo` does not change.

For example, move to Line 88,
```
if (parentBuildInfo && parentBuildInfo.buildName) {
   const parentRegex = /openjdk(\d+)-pipeline/i;
   const parenttokens = parentBuildInfo.buildName.match(parentRegex);
   jdkVersion = parenttokens[1];
}

```

Reply:
Sounds good, I originally had it outside of the forEach loop, not sure why I moved it. 👍